bin/run

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
+
+require('@oclif/command').run()
+.then(require('@oclif/command/flush'))
```

Comment:
Indentation

Reply:
this file is auto generated by oclif, but ill add it to the prettify and lint